mobile_app_connector/models/mobile_writing.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Emanuel Cino <ecino@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+import logging
+
+from odoo import api, models, fields, _
+from odoo.exceptions import ValidationError
+
+logger = logging.getLogger(__name__)
+
+
+class AppWriting(models.Model):
+    _name = 'mobile.app.writing'
+    _description = 'Mobile App Writing'
+    _order = 'state asc, print_count asc, date_start asc'
+
+    ##########################################################################
+    #                                 FIELDS                                 #
+    ##########################################################################
+    name = fields.Char('Category title', translate=True, required=True)
+    icon = fields.Char('FontAwesome icon name', required=True)
+    external_url = fields.Char(translate=True)
+    date_start = fields.Date(
+        readonly=True,
+        states={'new': [('readonly', False)]}
+    )
+    date_stop = fields.Date(
+        readonly=True,
+        states={'new': [('readonly', False)]}
+    )
+    is_active = fields.Boolean()
```

Comment:
```suggestion
    active = fields.Boolean()
```
We did it badly in other models, but `active` is a standard Odoo field which is there for that.